### ISSUE TITLE
Finalize logging analyzers

### ISSUE THREAD

**Issue by pakrym** [aspnet/Logging#712]
Consider adding analyzers for `DefineMessage`
Reference analyzers from our projects
Decide how are they going to be shipped

/cc @glennc

**Comment by paulomorgado** [comment]
Where are these analyzers published?

**Comment by pakrym** [comment]
Right now they are published to https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json feed. https://dotnet.myget.org/feed/aspnetcore-ci-dev/package/nuget/Microsoft.Extensions.Logging.Analyzers

**Comment by paulomorgado** [comment]
Will they work on VS2015?

**Comment by paulomorgado** [comment]
@pakrym, still no plans to release this?

**Comment by pakrym** [comment]
No, not in 2.1

**Comment by paulomorgado** [comment]
😢

**Comment by Eilon** [comment]
@pakrym - are we shipping them in "experimental" form so that people can easily try them out? I thought I heard someone say that the other day.

**Comment by pakrym** [comment]
Yes, we would continue to ship them in the same form they are now, but wouldn't do the same thing we did with MVC and EF analyzers (reference from packages themselves)

**Comment by Genbox** [comment]
I think these analyzers should be a little more extensive than they are now. It would also be great if they supported structured logging templates. The structured message analyzer rule can be 'none' severity by default and only enabled if users want them.
